File: services/reservation/src/main/java/com/example/iter/reservation/dto/response/ReturnComparisonResponse.java
package com.example.iter.reservation.dto.response;

import com.example.iter.auth.api.UserSummary;

import java.time.LocalDate;

public record ReturnComparisonResponse(
        Long rentalId,
        String equipmentName,
        UserSummary renter,
        LocalDate startDate,
        LocalDate endDate,
        LocalDate returnDate,
        ConditionEvidenceResponse receipt,
        ConditionEvidenceResponse returnReceipt
) {
}

File: services/reservation/src/main/kotlin/com/example/iter/reservation/dto/response/ReturnComparisonResponse.kt
package com.example.iter.reservation.dto.response

import com.example.iter.auth.api.UserSummary
import java.time.LocalDate
import kotlin.jvm.JvmRecord

@JvmRecord
data class ReturnComparisonResponse(
    val rentalId: Long?,
    val equipmentName: String?,
    val renter: UserSummary?,
    val startDate: LocalDate?,
    val endDate: LocalDate?,
    val returnDate: LocalDate?,
    val listingImages: List<ConditionEvidenceImageResponse> = emptyList(),
    val receipt: ConditionEvidenceResponse?,
    val returnReceipt: ConditionEvidenceResponse?
)
